docs/recipes/browserify-multiple-destination.md

```diff
@@ -0,0 +1,43 @@
+# Browserify + Globs (multiple destination)
+
+This example shows how to set up a task of bundling multiple entry points into multiple destinations using browserify.
+
+The below `js` task bundles all the `.js` files under `src/` as entry points and writes the results under `dest/`.
+
+
+```js
+var gulp = require('gulp');
+var plugins = require('gulp-load-plugins')();
```

Comment:
probably shouldnt use gulp-load-plugins in the recipes, just to keep them agnostic